ui/shadow_priest/presets.ts

```diff
@@ -60,9 +71,11 @@ export const GearPresets = {
 	[Phase.Phase4]: [GearPhase4],
 	[Phase.Phase5]: [GearPhase5Draconic, GearPhase5CoreForged],
 	[Phase.Phase6]: [GearPhase6],
+	[Phase.Phase7]: [PresetBuildPhase7.gear!],
+	[Phase.Phase8]: [PresetBuildPhase8.gear!],
 };
 
-export const DefaultGear = GearPresets[Phase.Phase6][0];
+export const DefaultGear = GearPresets[Phase.Phase8][0];
```

Comment:
You should be able to remove the `DefaultGear`, `DefaultTalents`, and `DefaultAPL` constants now and instead, in `sim.ts` pull directly from `DefaultBuild`! See the `ui/mage/sim.ts` for an example. I actually also fully removed buffs, debuffs, and consumes from the `presets.ts` in favor of pulling directly from the `DefaultBuild`

Reply:
Okay, I'll try to cleanup and align to the mage config.